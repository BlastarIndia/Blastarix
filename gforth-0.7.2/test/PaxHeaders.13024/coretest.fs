30 mtime=1360949212.220834245
30 atime=1361794994.396107769
30 ctime=1361794994.396107769
