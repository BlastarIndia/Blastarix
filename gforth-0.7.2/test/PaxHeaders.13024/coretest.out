30 mtime=1361394633.772976716
30 atime=1361794994.563111152
30 ctime=1361794994.563111152
